test/permission.validation.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const metatests = require('metatests');
+
+const {
+  setup,
+  prepareRemoteProvider,
+} = require('./fixtures/permission-validation/setup');
+
+const runTests = (test, ctx) => {
+  const testsDir = path.join(__dirname, 'permissions');
+
+  fs.readdir(testsDir, (err, testFiles) => {
+    if (err) {
+      console.error(
+        `Cannot read tests' directory, skipping permission validation tests.`
+      );
+      console.error(err);
+      test.end();
+      return;
+    }
+
+    for (const testFile of testFiles) {
+      const testFilePath = path.join(testsDir, testFile);
+      const subtest = require(testFilePath);
+      subtest(test, ctx);
+    }
+  });
+};
+
+metatests.test('Permission validation test', test => {
+  setup((err, ctx) => {
+    if (err) {
```

Comment:
Could you do the same as here please https://github.com/metarhia/globalstorage/pull/413/files#diff-a4aad6dd4b4793d911574d96490c433bR83? (Fail only on CI if postgres setup failed and only print the error not on CI)

Reply:
Done